src/components/ConversationsList/ConversationTile.tsx

```diff
@@ -1,36 +1,36 @@
 import { Divider, Flex, LinkBox, LinkOverlay, Text } from '@chakra-ui/react'
-import type { Conversation } from '@xmtp/xmtp-js'
-import React from 'react'
+import React, { useMemo } from 'react'
 import Emoji from 'react-emoji-render'
 import { formatDate, truncate } from '../../helpers'
-import useConversation from '../../hooks/useConversation'
+import useChat from '../../hooks/useChat'
 import Address from '../Address'
 import Avatar from '../Avatar'
 
 type ConversationTileProps = {
-  conversation: Conversation
+  peerAddress: string
   isSelected: boolean
   onClick?: () => void
 }
 
 // TODO: add design for `isSelected`
 const ConversationTile = ({
-  conversation,
+  peerAddress,
   isSelected,
   onClick,
 }: ConversationTileProps): JSX.Element | null => {
-  const { messages } = useConversation(conversation.peerAddress)
-
-  const latestMessage = messages.length ? messages[messages.length - 1] : null
-
+  const { store } = useChat()
+  const latestMessage = useMemo(() => {
+    const messages = store[peerAddress] || []
+    return messages[messages.length - 1]
+  }, [peerAddress, store])
```

Comment:
Any reason that this isn't using `useMessages(peerAddress)` instead?

```tsx
const messages = useMessage(peerAddress)
const latestMessage = messages[messages.length - 1]
```

Reply:
Just a missing update after the hook created but now done [c95a799](https://github.com/liteflow-labs/chat/pull/15/commits/c95a7991a9cc2f0b65e816402572badeeb333b22)